client/dashboard/sites/settings/site-actions.tsx

```diff
@@ -0,0 +1,82 @@
+import { useMutation } from '@tanstack/react-query';
+import { __experimentalHeading as Heading, Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { addQueryArgs } from '@wordpress/url';
+import { restoreSitePlanSoftwareMutation } from '../../app/queries';
+import { ActionList } from '../../components/action-list';
+import { DotcomFeatures } from '../../data/constants';
+import type { Site } from '../../data/types';
+
+const useRestorePlanSoftware = ( { slug, options }: Site ) => {
+	const mutation = useMutation( restoreSitePlanSoftwareMutation( slug ) );
+
+	if ( ! options?.is_wpcom_atomic ) {
```

Comment:
Can we use `site.is_wpcom_atomic` instead? I think it's more reliable than raw options, and is symmetrical with `is_wpcom_staging_site`. We'd need to pass it to the `SITE_FIELDS` first.

Reply:
Ok. But I noticed that both are used across Calypso 😓